mysql-to-kv/function.yaml

```diff
@@ -0,0 +1,53 @@
+
+# Copyright 2017 The Nuclio Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+apiVersion: "nuclio.io/v1beta1"
+kind: "Function"
+spec:
+  description: "A simple v3io-webapi usage for PutItem using MySQL public DB"
+  handler: "main:handler"
+  env:
+    - type: value
```

Comment:
1. No need for `type`
2. These values need to be passed as variables. See the values file in https://github.com/nuclio/nuclio-templates/tree/master/string-manipulator